docs/editor/extension-gallery.md

```diff
@@ -232,14 +232,13 @@ An example `extensions.json` could be:
 ```json
 {
     "recommendations": [
-        "ms-vscode.vscode-typescript-tslint-plugin",
         "dbaeumer.vscode-eslint",
         "msjsdiag.debugger-for-chrome"
     ]
 }
 ```
 
-which recommends two linter extensions, TSLint and ESLint, as well as the Chrome debugger extension.
+which recommends two linter extensions: ESLint, and the Chrome debugger extension.
```

Comment:
"which recommends a linter extension, ESLint, and the Chrome debugger extension."